packages/core/src/any_props.rs

```diff
@@ -65,7 +65,12 @@ unsafe impl<'a, P> AnyProps<'a> for VProps<'a, P> {
 
         match res {
             Ok(Some(e)) => RenderReturn::Ready(e),
-            _ => RenderReturn::default(),
+            Ok(None) => RenderReturn::default(),
+            Err(err) => {
+                let component_name = std::any::type_name::<P>();
```

Comment:
This will render the props name, not the component name. Usually the same, but not always. Every component has a name though, available via scopestate.

<img width="530" alt="Screenshot 2023-07-21 at 2 17 31 PM" src="https://github.com/DioxusLabs/dioxus/assets/10237910/9116e673-7a69-4a36-b7de-9efa946d2fbe">